src/DataTable/DataTable.stories.tsx

```diff
@@ -29,19 +29,19 @@ const defaultSale = {
 
 const originalData: Sale[] = [
   {
-    date: format(new Date(), 'MM-dd-yyyy'),
+    date: format(new Date('08-27-2021'), 'MM-dd-yyyy'),
```

Comment:
I think that when we declare default values is a good practice to replace them with a constant.

```suggestion
    const date = '08-27-2021';
    date: format(new Date(date), 'MM-dd-yyyy'),
```